pkg/proc/breakpoints.go

```diff
@@ -447,6 +449,10 @@ func evalBreakpointCondition(tgt *Target, thread Thread, cond ast.Expr) (bool, e
 			return true, err
 		}
 	}
+	scope, err = ConvertEvalScope(tgt, scope.g.ID, int(frameCond), 0)
```

Comment:
This is going to break if the thread isn't running a goroutine. This could be the reason for all the new test failures.